Source/WebKit/WebProcess/GPU/media/SourceBufferPrivateRemote.cpp

```diff
@@ -355,15 +355,24 @@ void SourceBufferPrivateRemote::setAppendWindowEnd(const MediaTime& appendWindow
     gpuProcessConnection->connection().send(Messages::RemoteSourceBufferProxy::SetAppendWindowEnd(appendWindowEnd), m_remoteSourceBufferIdentifier);
 }
 
-void SourceBufferPrivateRemote::seekToTarget(const WebCore::SeekTarget& target, CompletionHandler<void(const MediaTime&)>&& completionHandler)
+void SourceBufferPrivateRemote::computeSeekTime(const WebCore::SeekTarget& target, CompletionHandler<void(const MediaTime&)>&& completionHandler)
 {
     auto gpuProcessConnection = m_gpuProcessConnection.get();
     if (!isGPURunning()) {
         completionHandler(MediaTime::invalidTime());
         return;
     }
 
-    gpuProcessConnection->connection().sendWithAsyncReply(Messages::RemoteSourceBufferProxy::SeekToTarget(target), WTFMove(completionHandler), m_remoteSourceBufferIdentifier);
+    gpuProcessConnection->connection().sendWithAsyncReply(Messages::RemoteSourceBufferProxy::ComputeSeekTime(target), WTFMove(completionHandler), m_remoteSourceBufferIdentifier);
+}
+
+void SourceBufferPrivateRemote::seekToTime(const MediaTime& time)
+{
+    auto gpuProcessConnection = m_gpuProcessConnection.get();
```

Comment:
We could get it after the isGPURunning check

Reply:
we could, but there are heaps of methods doing the same. would make a too big diff